pdns/recursordist/docs/settings.rst

```diff
@@ -1224,6 +1224,19 @@ it by the number of NS records found above the
 `max-ns-address-qperq`_ value. The limit wil not be reduced to a
 number lower than 5.
 
+.. _setting-max-nsperq:
```

Comment:
A small nit, I find the `perq` suffix confusing for this setting, as it suggests (to me, at least) that it is the number of NS we are willing to consider per query, while it's actually the number of NS we are considering per zone?

Reply:
It's wrong indeed. It's per name, I'd say. Once client query might need to resolve several names, each such resolve is subject to the restriction, while max-qperq is _per client query_.

Would the name `max-ns-per-resolve` suit you better? Cannot think of a better name atm.

Reply:
Yes, it would be much better. Thanks!